make/Make.config

```diff
@@ -120,12 +120,7 @@ ifndef SDKBUILD
 SDKBUILD :=build
 endif
 
-ifdef SDK_OUTDIR
```

Comment:
these changes were added by brcm in 6.1.0.3.. was this removed in 7.0.0.6 or is this our change?

Reply:
Good catch. Fixed it. Thanks.